metrics/grafana/pd.json

```diff
@@ -11369,7 +11369,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(delta(pd_scheduler_region_heartbeat{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", store=~\"$store\", type=\"report\", status=\"err\"}[1m])) by (address, store)",
+              "expr": "sum(rate(pd_scheduler_region_heartbeat{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", store=~\"$store\", type=\"report\", status=\"err\"}[1m])) by (address, store)",
```

Comment:
need change L11279 also？

Reply:
I think we don't change it. We have multiple 60 times in L11279, so we should use opm.

Reply:
I mean that we can change it and don't multiply 60. So all expressions will use `ops`